Makefile

```diff
@@ -75,24 +71,28 @@ version.h: Makefile .git/index
 .git/index:
 
 clean:
-	$(RM) *.o nsxiv
+	$(RM) *.o nsxiv version.h lib_autodetect.mk
 
 install: all
 	@echo "INSTALL bin/nsxiv"
-	install -Dt $(DESTDIR)$(PREFIX)/bin nsxiv
+	mkdir -p $(DESTDIR)$(PREFIX)/bin
+	cp nsxiv $(DESTDIR)$(PREFIX)/bin/
+	chmod 755 $(DESTDIR)$(PREFIX)/bin/nsxiv
```

Comment:
Why use 3 commands instead of 1?

Reply:
I think I explained that in the OP.

> Also revert back to using mkdir -> cp -> chmod since install is not posix.

Reply:
That you did. My bad. I had thought install was POSIX.